Wikipedia/Code/WKWebView+LoadAssetsHtml.swift

```diff
@@ -0,0 +1,96 @@
+import WebKit
+import WMF
+
```

Comment:
@joewalsh heads-up I refactored `WKWebView+LoadAssetsHtml.m` to Swift.

I don't think it needs to call `stringByReplacingImageURLsWithProxyURLs` any more so I removed the call. See comments on this commit: https://github.com/wikimedia/wikipedia-ios/pull/1890/commits/bd9140615f1dc915f423ecaf643a0f7e556e29df